test/unit/specs/main.spec.js

```diff
@@ -244,12 +251,12 @@ describe("Startup Process", () => {
       expect(fs.existsSync(testRoot)).toBe(true)
     })
 
-    xit("should init lcd server with correct testnet", async function() {
+    it("should init lcd server with correct testnet", async function() {
```

Comment:
is the test description intended like this?

Reply:
You're right, that test was intended for initting the data directory and not starting the LCD. I added the testnet check to the LCD start test instead.